frontend/public/components/utils/log-window.jsx

```diff
@@ -124,17 +106,19 @@ export class LogWindow extends React.PureComponent {
           </div>
         </div>
       </div>
-      { !['streaming', 'loading', 'eof'].includes(this.props.status) && <div onClick={this._unpause} className="log-window__footer">
-        { !hasLinesBehind && <div><span className="fa fa-play-circle-o"></span> Resume stream</div> }
-        { hasLinesBehind && linesBehind < this.props.buffer.maxSize && <div><span className="fa fa-play-circle-o"></span> Resume stream and show {pluralize(linesBehind, 'line')}</div> }
-        { hasLinesBehind && linesBehind > this.props.buffer.maxSize && <div><span className="fa fa-play-circle-o"></span> Resume stream and show last {pluralize(this.props.buffer.maxSize, 'line')}</div> }
+      { !['streaming', 'loading', 'eof'].includes(status) && <div onClick={this._unpause.bind(this)} className="log-window__footer">
```

Comment:
Not new, but this isn't accessible. (We don't necessarily have to fix in this PR. Just noting it. Would be good to address in a follow-on.)

Reply:
Should we create an issue for this so it doesn't get lost?